docs/storagenetwork/storagenetwork.md

```diff
@@ -0,0 +1,239 @@
+
+For more informations, please refer [Longhorn Storage Network](https://longhorn.io/docs/1.3.2/advanced-resources/deploy/storage-network/)
+
+:::note
+
+We always recommend using Harvester Storage Network setting instead of Longhorn Storage Network setting directly. Manual configuration in the Longhorn setting might confuse the determinations of the Harvester controller.
+
+:::
+
+## Prerequisites
+
+There are some prerequisites before configuring the Harvester Storage Network setting.
+
+- Well-configured Cluster Network and VLAN Config
+    - Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes. 
+- All VMs should be stopped.
+    - We recommend checking the VM status with the following command and should get an empty result.
+    - `kubectl get -A vmi`
+- All pods that are attached to Longhorn Volumes should be stopped.
+    - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
+
+## Configuration Example
+
+- IP range for Storage Network
+	- IP range should not conflict or overlap with Kubernetes cluster networks.
+	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
+	- If your cluster has 250 nodes, the IP range should be larger than /22.
+- Well-configured Cluster Network and VLAN Config
+	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
+- VLAN ID
+	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
+
+We will take the following configuration as an example to explain the details of the Storage Network
+
+- IP range: `192.168.0.0/24`
+- Cluster Network: `storage`
+- VLAN ID for Storage Network: `100`
+
+## Configuration Process
+
+Harvester will create Multus NetworkAttachmentDefintion from the configuration, stop pods related to Longhorn Volume, update Longhorn setting, and restart previous pods.
+
+### Before Applying Harvester Storage Network Setting
+
+Here we have two cases.
+- Expect that VM VLAN traffic and Longhorn Storage Network use the same group of physical interfaces.
+- Expect that VM VLAN traffic and Longhorn Storage Network use different physical interfaces.
+
+Longhorn will send replication traffic through the specific interfaces shown as the red line in the figure.
+
+#### Same Physical Interfaces
+
+Take `eth2` and `eth3` as an example for VM VLAN traffic and Longhorn Storage Network simultaneously.
+
+Please refer Networking page to configure `ClusterNetwork` and `VLAN Config` with `eth2` and `eth3` and remember the `ClusterNetwork` name for the further step.
+
+![storagenetwork-same.png](./storagenetwork-same.png)
```

Comment:
since this is a bond so both `eth2` and `eth3` could be used by the storage network instead `eth3` only?

Reply:
Both will be used.
It depends on bond mode.
let me update figures for bonding

Reply:
update figure with bonding.
this should not confuse again